plugins/notifiers/slack/client.go

```diff
@@ -67,11 +67,16 @@ func (n *notifier) Notify(items []domain.Notification) []error {
 	return errs
 }
 
-func (n *notifier) sendMessage(channel, text string) error {
+func (n *notifier) sendMessage(channel, block string) error {
```

Comment:
would suggest to keep the variable name something like `messageBlock` or `message`. `block` as standalone word doesn't convey much in the func name.

Reply:
Done